src/snapred/backend/recipe/algorithm/RawVanadiumCorrection.py

```diff
@@ -0,0 +1,219 @@
+import json
+from typing import Any, Dict, Tuple
+
+import numpy as np
+from mantid.api import (
+    AlgorithmFactory,
+    ITableWorkspaceProperty,
+    MatrixWorkspaceProperty,
+    PropertyMode,
+    PythonAlgorithm,
+)
+from mantid.kernel import Direction
+from scipy.interpolate import make_smoothing_spline, splev
+
+from snapred.backend.dao.ingredients import ReductionIngredients as Ingredients
+from snapred.backend.dao.state.CalibrantSample.CalibrantSamples import CalibrantSamples
+from snapred.backend.recipe.algorithm.MakeDirtyDish import MakeDirtyDish
+from snapred.backend.recipe.algorithm.MantidSnapper import MantidSnapper
+
+name = "RawVanadiumCorrection"
+
+
+# TODO: Rename so it matches filename
+class RawVanadiumCorrection(PythonAlgorithm):
```

Comment:
I'd prefer if algo names were appended with `Algo` at least.  We dont adhere to this everywhere, but I'd like to try.

Reply:
Okie dokie